src/nodemetadata.cpp

```diff
@@ -18,13 +18,10 @@
 */
 
 NodeMetadata::NodeMetadata(IItemDefManager *item_def_mgr):
-	m_inventory(new Inventory(item_def_mgr))
+	m_inventory(std::make_unique<Inventory>(item_def_mgr))
 {}
 
-NodeMetadata::~NodeMetadata()
-{
-	delete m_inventory;
-}
+NodeMetadata::~NodeMetadata() = default;
```

Comment:
this should be in the header and not here

Reply:
No. If in header, you'd have to include inventory.h there. This is better.

Reply:
right.